source/_integrations/solarlog.markdown

```diff
@@ -34,25 +38,41 @@ Please note that the open JSON interface only exposes a limited amount of data.
 
 {% include integrations/config_flow.md %}
 
-## Additional template sensor
+To setup the integration you need the following information:
 
-In case you would like to get additional calculated sensors such as the amount of excess solar power available, you can use the [template platform](/integrations/template/).
+{% configuration_basic %}
+Name:
+    description: "The name for your Solar-Log device in Home Assistant. This will also be uses as prefix for the entity names."
+    required: true
+    type: string
```

Comment:
`configuration_basic` does not have support for these parameters.

```suggestion
```

Reply:
Took this from the [documentation](https://developers.home-assistant.io/docs/core/integration-quality-scale/rules/docs-installation-parameters/), this should then be amended...